app/views/content_change_requests/_request_details.html.erb

```diff
@@ -181,7 +181,11 @@ This is some content that has changed.
 <%= render partial: "support/time_constraint", locals: { f: f } %>
 
 <div class="alert alert-info alert-block">
-  <p>If your request is urgent, please fill in this form and call 07810 750 834 during office hours, or 07827 992603 out of hours. (It's ONLY urgent when the public or the government is facing immediate and significant financial, legal or physical risk.)</p>
+  <p><strong>If your request is urgent</strong></p>
+  <p>Submit this form and call:</p>
+  <ul>
+    <li><a href="tel:+447810750834">07810 750 834</a> between 9am and 6pm</li>
+    <li><a href="tel:+447827992603">07827 992 603</a> at other times</li>
```

Comment:
Looks like you're missing "It's urgent only if the public or the government faces immediate and significant financial, legal or physical risk".

Reply:
This is still a thing @DilwoarH ^